packages/python-packages/api-stub-generator/apistub/nodes/_argtype.py

```diff
@@ -0,0 +1,54 @@
+import logging
+
+# Special default values that should not be treated as string literal
+SPECIAL_DEFAULT_VALUES = ["None", "..."]
+
+
+class ArgType:
+    """Represents Argument type
+    :param str: name
+        Name of the argument
+    :param str: argtype
+        Type of the argument. for e.g. str, int, BlobBlock
+    :param str: default
+        Default value for the argument, If any
+    """
+
+    def __init__(self, name, argtype=None, default=None):
+        super().__init__()
+        self.argname = name
+        self.argtype = argtype
+        self.default = default
+
+    def generate_tokens(
+        self, apiview, function_id, add_line_marker, include_default=True
+    ):
+        """Generates token for the node and it's children recursively and add it to apiview
+        :param ApiView: apiview
```

Comment:
This format isn't correct, should be like:
`:param ~ApiVersion apiview: The ApiView`